libpod/oci.go

```diff
@@ -749,7 +749,11 @@ func (r *OCIRuntime) execContainer(c *Container, cmd, capAdd, env []string, tty
 
 	args = append(args, "exec")
 
-	args = append(args, "--cwd", c.config.Spec.Process.Cwd)
+	if cwd != "" {
+		args = append(args, "--cwd", cwd)
+	} else {
```

Comment:
I think we can leave the `else` out here - runc should use whatever is already in the spec in that case automatically

Reply:
Done.